fe/src/main/java/org/apache/doris/qe/GlobalVariable.java

```diff
@@ -56,6 +56,11 @@
     @VariableMgr.VarAttr(name = "query_cache_size")
     private static long queryCacheSize = 1048576;
 
+    public static final String HEARTBEAT_FLAGS = "heartbeat_flags";
```

Comment:
`public static final String HEARTBEAT_FLAGS = "heartbeat_flags";`

move this line to the above, with other variables' name declaration.

Reply:
remove it